include/deal.II/grid/tria_info_cache_flags.h

```diff
@@ -0,0 +1,133 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_grid_tria_info_cache_flags_h
+#define dealii_grid_tria_info_cache_flags_h
+
+
+#include <deal.II/base/config.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+   * The enum type given to the TriangulationInfoCache class to select what
+   * information to cache.
+   *
+   * You can select more than one flag by concatenation using the bitwise or
+   * <code>operator|(TriangulationInfoCacheFlags,TriangulationInfoCacheFlags)</code>.
+   *
+   * @author Luca Heltai, 2017.
+   */
+enum TriangulationInfoCacheFlags
+{
+  /**
+   * Cache Nothing.
+   */
+  cache_nothing = 0,
+
+  /**
+   * Cache vertex_to_cell_map, as returned by GridTools::vertex_to_cell_map().
+   */
+  cache_vertex_to_cell_map = 0x0001,
+};
+
+
+/**
+   * Output operator which outputs assemble flags as a set of or'd text values.
+   *
+   * @ref TriangulationInfoCacheFlags
+   */
+template <class StreamType>
+inline
+StreamType &operator << (StreamType &s, TriangulationInfoCacheFlags u)
+{
+  s << " TriangulationInfoCacheFlags";
+  if (u & cache_vertex_to_cell_map) s << "|vertex_to_cell_map"        ;
+  return s;
+}
+
+
+/**
```

Comment:
Same here and all other places below.

Reply:
It think this is fixed, but github doesn't know it... :)